compiler/src/main/kotlin/io/spine/protodata/renderer/SourceFileSet.kt

```diff
@@ -41,7 +41,7 @@ import kotlin.text.Charsets.UTF_8
 /**
  * A set of source files.
  */
-public class SourceSet
+public class SourceFileSet
```

Comment:
If this rename is due to a clash with Gradle, I don't think we should sacrifice brevity. Gradle `SourceSet` and ProtoData `SourceSet` live in different contexts and barely (if ever) meet.

If there is another reason, please, tell me.

Reply:
`SourceSet` from Gradle is a thing bigger than what we have. It's more than just `Set<SourceFile>`. If we “reuse” the name, the user of ProtoData and `tool-base` APIs should be surprised less. Usages of the class are way wider than its package, and chances that would be usage scenarios where `SourceSet` and `SourceFileSet` would be used in a context of a single user class.